common/analysis/lint_rule_status.h

```diff
@@ -28,27 +28,101 @@
 #include "common/text/symbol.h"
 #include "common/text/syntax_tree_context.h"
 #include "common/text/token_info.h"
+#include "common/util/logging.h"
```

Comment:
Don't forget to add `//common/util/logging` to the dependencies of lint_rule_status in the BUILD file.

Reply:
done